### PR TITLE
Update the rules

### PR DESCRIPTION
**Pull request by TimCastelijns**

### CODE REVIEW

File: body.md [hunk 1]
```diff
@@ -27,31 +28,33 @@ You can request write access once you've joined the room. When a room owner is a
 
 <section markdown='1'>
 
-1. Before you ask a question here, first [thoroughly](http://stackoverflow.com/search) [research](http://programmers.stackexchange.com/search) [it](https://www.google.com) [yourself](http://www.ietf.org/rfc.html), and post it on [Stack Overflow](http://stackoverflow.com/) (or another appropriate Stack Exchange site, like [programmers.se](http://programmers.stackexchange.com/)). We also expect you to read the [Android developer site](http://developer.android.com/) before asking a question about Android.
+1. Before you ask a question here we expect you to have done your own research on the matter. This includes but is not limited to having searched on Stack Overflow, android developer documentation and search engines. If the applicable, we also expect you to have asked the question on Stack Overflow.
```
Comment (davebren) [inline body.md]: If the applicable, --> If applicable

File: body.md [hunk 2]
```diff
@@ -27,31 +28,33 @@ You can request write access once you've joined the room. When a room owner is a
...
 
-2. Use English and try to write well. This means no chat-speak (e.g., "u r vry gud i m new dev lol").
+2. Communicate in English and to the best of your ability. We don't expect everybody to master the language but we do not want to see chat-speak (e.g. "u r vry gud i m new dev lol").
```
Comment (davebren) [inline body.md]: Communicate in English _to_ the best of your ability.

Comment (davebren) [reply body.md]: We don't expect everyone*

File: body.md [hunk 3]
```diff
@@ -27,31 +28,33 @@ You can request write access once you've joined the room. When a room owner is a
...
 
-3. Don't paste blocks of code into the chat. Make a [gist] or use another pastebin if you must share code. Any code in the chatroom should be limited to one line at about 80 columns max.
+3. Make a [gist] or use a similar tool if you want to share code. Any code posted in the room should fit in one or two lines.
```
Comment (davebren) [inline body.md]: fit on* one or two lines

File: body.md [hunk 4]
```diff
@@ -27,31 +28,33 @@ You can request write access once you've joined the room. When a room owner is a
...
 
 6. Your first message must be "I have read and understood the rules".
 
-7. Images MUST be safe for work. Please don't post unprofessional images. Kitties and unicorns: okay. A screenshot of your app: sure. Twerking: no.
+7. Don't flag messages based on the mere presence of certain words (swearing, blasphemy, etc) which are often seen in casual language. Chat message flags raise a site-wide notification for all users with 10000+ rep, which can be annoying, especially if a flag was not really warranted.  
+Do note that while we have this policy, we do not want to discourage anyone from flagging anything they feel violates Stack Overflow's Code of Conduct or [Be Nice] policy. See [here](https://meta.stackexchange.com/help/flagging) for help with that.
 
-8. Don't flag messages you find personally offensive (e.g., swear words, blasphemy, and so on). It just annoys the people who have to respond to flags.
+8. Only codeMagic is allowed to buy and drink [ass-Bourbon](http://sourmashmanifesto.com/2012/02/01/review-rebel-yell-bourbon/). Everyone else should [know better](http://chat.stackoverflow.com/transcript/message/19483807#19483807).
```
Comment (rashiq) [inline body.md:46]: should we retire this?

File: body.md [hunk 5]
```diff
@@ -27,31 +28,33 @@ You can request write access once you've joined the room. When a room owner is a
...
-4. Don't ping users (@username-ing) unless it's to clarify who you're speaking to or you need to get a specific person's attention. Pinging for help is bannable.
+4. Don't ping users (@username-ing) unless it's to clarify who you're speaking to or you need to get a specific person's attention.
 
-5. Everything kitty-related is allowed.
+5. All posted content must be safe for work. If you want to share an image or a link that is not safe for work, hide the content in a hyperlink and include NSFW in the title (also do this if you are not sure if it is safe). The markup for such a hyperlink is `[Check this out (NSFW)](https://example.org)`.
 
 6. Your first message must be "I have read and understood the rules".
 
-7. Images MUST be safe for work. Please don't post unprofessional images. Kitties and unicorns: okay. A screenshot of your app: sure. Twerking: no.
+7. Don't flag messages based on the mere presence of certain words (swearing, blasphemy, etc) which are often seen in casual language. Chat message flags raise a site-wide notification for all users with 10000+ rep, which can be annoying, especially if a flag was not really warranted.
```
Comment (rashiq) [inline body.md:43]: looks like it's a bit too long, isn't it?

Comment (TimCastelijns) [reply body.md:43]: Not really. We had a mod coming into the room last week making sure we are not sending the wrong message with this rule. The first sentence is more or less what he suggested we change it into, I added the other sentence to explain the first.

File: body.md (hunk 4 above)
Comment (TimCastelijns) [reply body.md:46]: I wanted to remove it, but scared for backlash.

**Review by MobileOak — APPROVED** [review]

File: body.md (hunk 4 above)
Comment (AdamMc331) [reply body.md:46]: It's a staple. I vote for keeping it, or replacing it with a modern joke like being tired of someone's shit.

Comment (AdamMc331) [reply body.md:46]: Gotta have a little humor in there, I don't want our rules to make us sound stuck up.

**Comment by AdamMc331** [comment]
I see Carl reviewed this, but I'm in favor too. Thanks Tim!

File: body.md (hunk 4 above)
Comment (Mauker1) [reply body.md:46]: I vote for replacing this for: Only McAdam331 is allowed to lose his keystore.

Comment (ankitagrawal2411) [reply body.md:46]: lmao

**Review by rashiq — APPROVED** [review]

File: body.md (hunk 5 above)
Comment (RaghavSood) [reply body.md:43]: I don't think length is a problem, it's not too much of an ask to have someone read a few sentences.

**Pull request merged**